src/api/user/routes.js

```diff
@@ -0,0 +1,15 @@
+const { Router } = require("express");
+const controller = require("./controllers");
+const { authenticateToken } = require("../auth/middlewares");
+const { isAdmin } = require("../user/middlewares");
+const route = Router();
+
+module.exports = (app) => {
+  app.use("/users", route);
+
+  route.get("/", authenticateToken, controller.getAllUsers);
```

Comment:
Just one question...Who can see all users? Just those who have admin role or everyone? @kimnovak

Reply:
Good question, it should be admin only

Reply:
I will add isAdmin middleware then, thanks for noticing that @bojanab987